ci/jenkins/Nightly.groovy

```diff
@@ -141,6 +144,8 @@ pipeline {
                                                     --set queryNode.replicas=2 \
                                                     --set indexNode.replicas=2 \
                                                     --set dataNode.replicas=2 \
+                                                    --set mysql.enabled=true \
```

Comment:
Add to kafka only, not all pipelines. Please modify it. Thanks.

Reply:
> Add to kafka only, not all pipelines. Please modify it. Thanks.

Modified.